packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -648,8 +648,8 @@ const Slider = React.forwardRef(function Slider(props, ref) {
         const percent = valueToPercent(mark.value, min, max);
         const style = axisProps[axis].offset(percent);
         const markActive = range
-          ? percent >= values[0] && percent <= values[values.length - 1]
-          : percent <= values[0];
+          ? mark.value >= values[0] && mark.value <= values[values.length - 1]
+          : mark.value <= values[0];
```

Comment:
I think this is an actual bug fix. At least from looking at it it makes more sense that way.

Reply:
https://github.com/mui-org/material-ui/issues/16247#issuecomment-502378379